### DISCUSSION TITLE
supabase start error with invalid peer certificate

### DISCUSSION THREAD

**Discussion by asyarifstudio** [supabase/supabase#21106] [Questions]
I've been having this issue when executing `supabase start`

S C:\work\temporary\Youcut\youcut-server> supabase start
You are running outdated service versions locally:
supabase/postgres:15.1.0.147 => 15.1.1.5
Run supabase link to update them.
supabase_edge_runtime_Youcut container logs:
                                            Download https://deno.land/std@0.182.0/http/server.ts
                                                                                                 Download https://deno.land/x/jose@v4.13.1/index.ts
        failed to load 'https://deno.land/std@0.182.0/http/server.ts': Import 'https://deno.land/std@0.182.0/http/server.ts' failed: error 
sending request for url (https://deno.land/std@0.182.0/http/server.ts): error trying to connect: invalid peer certificate: UnknownIssuer   
                                                                                                                                        Error: main worker boot error: worker boot error

I'm running on windows 11 machine.

I look online and some people have the same problem like this https://github.com/orgs/supabase/discussions/10989. but the solution to set DENO_CERT to a certificate file path. Where do I get this certificate? is it something download from somewhere or you have to generate it.

I've been trying to find for last 3 days without any clue.

**Comment by GaryAustin1** [comment]
I don't use this, but this comment implies from the link to your database section in the dashboard.  Sorry if wrong info.
![image](https://github.com/supabase/supabase/assets/54564956/f5aee2c5-a2b3-4115-abbe-095f3e658bae)
https://supabase.com/docs/guides/functions/connect-to-postgres#ssl-connections

**Reply by asyarifstudio** [reply]
I use this certificate and still unable to get it running. I combine it also with solution in [10989](https://github.com/orgs/supabase/discussions/10989) and it also not running.

**Comment by caseybaggz** [comment]
Same here with the same problem of it being very hard to find the resolution to this error that actually provides good context on how to solve it. 😞

**Comment by asyarifstudio** [comment]
I finally resolve it. The issue, in my case is not with the supabase but with docker. I use corporate laptop that intercept the SSL layer by adding a a new certificate in the chain, https://serverfault.com/a/1100505 for more details

So I connect my laptop to corporate VPN and set the proxy in the docker. I did it once and after that I dont see this issue anymore. I'm not sure why but I guess the docker does not trust the Corporate Root CA, but after using proxy once, it register this certificate as trusted certificate. may be

**Comment by JBhrayn** [comment]
@asyarifstudio I've been having the same problem, but this particularly occurs when I don't have an internet connection. Do I really need to have internet connection to be able to run an edge runtime container or do I have to setup something else to get it working? 
![image](https://github.com/supabase/supabase/assets/110586557/867214c8-a519-4b09-be19-0dc3c8911409)

**Comment by JayMGurav** [comment]
any updates with the above issue??

**Comment by emptytank** [comment]
Been having the same issue, anyone resolve this?
